src/target/stm32h7.c

```diff
@@ -404,7 +404,10 @@ static bool stm32h7_flash_erase(target_flash_s *const target_flash, target_addr_
 	if (!stm32h7_flash_unlock(target, addr))
 		return false;
 	/* We come out of reset with HSI 64 MHz. Adapt FLASH_ACR.*/
-	target_mem_write32(target, flash->regbase + FLASH_ACR, 0);
+	uint32_t acr = 0;
+	/* H7Bx starts up in VOS3 from HSI 64 MHz with default ACR of 0x13 */
+	if (sector_size == 0x2000U) acr = 0x13;
```

Comment:
Can do. By the way, do you think H74x will benefit from increasing their ACR to 0x11 or all the way to 0x37 wrt stability/correctness? Or I could just always write 0x13 for all families if only it worked (less code complexity, less flash space)

Reply:
Without looking in the TRM, we don't recall what 0x37 and 0x11 for that control value mean - but if you think it will prevent the micro getting destabilised during Flash programming then we see no harm. We do wonder why the code was modifying a control value like that rather than respecting w/e the user has set already.

Reply:
Let's just say H7Bx can't practically operate with an ACR of 0 (under a running firmware or after reset). Whether touching it is required at all -- is a better question, and I leave it to you and other part users to test.